scripts/process_asr_text_tokenizer.py

```diff
@@ -110,6 +113,13 @@
     help="Samples the dataset by `sample_size` if positive integer, otherwise uses whole dataset",
 )
 parser.add_argument('--spe_train_extremely_large_corpus', action='store_true', help='')
+parser.add_argument(
+    '--spe_max_sentencepiece_length',
+    type=int,
+    default=-1,
+    help='Limit the maximum number of tokens in each subword. '
```

Comment:
Limit the maximum number of tokens in each subword  for sentence piece tokenizer.